appveyor.yml

```diff
@@ -24,21 +24,6 @@ test_script:
   - node --version && npm --version
   - echo -- BABEL 7 --
```

Comment:
this echo is redundant now, as we are testing only against babel 7, you should remove it